docs/source/platform/schema-registry.mdx

```diff
@@ -12,9 +16,12 @@ Apollo includes a schema registry that serves as a [central hub](https://princip
 - Having a registry allows you to disable introspection in production – a recommended best practice for good security.
 - Tools like the [Apollo VS Code extension](https://marketplace.visualstudio.com/items?itemName=apollographql.vscode-apollo) can automatically fetch your schema from the registry and provide intellisense like field descriptions and deprecations directly in your editor.
 - Apollo's registry lets you track related _variants_ of a schema, like staging or alpha versions. It's helpful to have these schema definitions handy without having to juggle running servers that implement them.
+- For a [federated](https://www.apollographql.com/docs/apollo-server/federation/introduction/) graph, Apollo supports registering individual services that make up your graph and associating them with partial schemas. This separates ownership across implementing services and allows you to track the composition of your graph. See below for details on registering federated graphs. 
 
 ## Using the Schema Registry
 
+// TODO: [Wherever the setup is for `apollo.config.js`, the assignee should link to it here and update the content there with any changes necessary]
```

Comment:
I need info on this, I don't know if any changes need to be made, but since we have the --endpoint tag I don't think they do